io.github.nemuelw.proton-desktop.metainfo.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Copyright 2024 Nemuel Wainaina -->
+<component type="desktop-application">
+  <id>io.github.nemuelw.proton-desktop</id>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <name>Proton Desktop</name>
+  <icon type="stock">io.github.nemuelw.proton-desktop.png</icon>
+  <summary>Unofficial desktop app for Proton</summary>
+  <description>
+    <p>
+      Proton Desktop is a free open-source unofficial desktop app for Proton.
+      It brings together Proton Mail, Proton Calendar, and Proton Drive into a single Linux application.
+    </p>
+  </description>
+
+  <developer id="io.github.nemuelw">
+    <name>Nemuel Wainaina</name>
+  </developer>
+
+  <launchable type="desktop-id">io.github.nemuelw.proton-desktop.desktop</launchable>
+  <content_rating type="oars-1.1" />
+  <branding>
+    <color type="primary" scheme_preference="light">#2a2c5e</color>
+    <color type="primary" scheme_preference="dark">#2a2c5e</color>
+  </branding>
+
+  <url type="homepage">https://github.com/nemuelw/proton-desktop</url>
+  <url type="donation">https://github.com/nemuelw/proton-desktop#donations</url>
+  <url type="vcs-browser">https://github.com/nemuelw/proton-desktop</url>
+  <url type="bugtracker">https://github.com/nemuelw/proton-desktop/issues</url>
+
+  <categories>
+    <category>Network</category>
+    <category>Utility</category>
+    <category>Email</category>
+    <category>Calendar</category>
+    <category>Archiving</category>
+  </categories>
+
+  <keywords>
+    <keyword>Proton Mail</keyword>
+    <keyword>Proton Calendar</keyword>
+    <keyword>Proton Drive</keyword>
+  </keywords>
+
+  <screenshots>
+    <screenshot url="https://github.com/nemuelw/proton-desktop/blob/main/scrshots/loading.png"/>
+    <screenshot url="https://github.com/nemuelw/proton-desktop/blob/main/scrshots/mail.png"/>
+    <screenshot url="https://github.com/nemuelw/proton-desktop/blob/main/scrshots/calendar.png"/>
+    <screenshot url="https://github.com/nemuelw/proton-desktop/blob/main/scrshots/drive.png"/>
+  </screenshots>
+
```

Comment:
where did you see this format?

Reply:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#screenshots

Reply:
and the links should be direct links to the image, not links to HTML pages.

https://raw.githubusercontent.com/nemuelw/proton-desktop/954ac083884bacab5aab7c2c535e6bc4b15ee9c6/scrshots/loading.png